options.go

```diff
@@ -70,7 +99,7 @@ func (opts *Options) lookupEnv(key string) (string, bool) {
 }
 
 func loadSources(opts *Options) error {
-	if !opts.Archive.Ignore && (len(opts.Archive.Contents) > 0 || len(opts.Archive.Name) > 0) {
+	if !opts.Archive.Ignore && !opts.Archive.IsEmpty() {
```

Comment:
Also, maybe a side-question for Ivan/Pablo, why the `loadArchive` is uniq the way we load, I mean it seems like if archive provided we completely ignore other paths, like manifest/env, could it cause any side effects? :thinking: